.eslintrc.js

```diff
@@ -130,11 +139,15 @@ module.exports = {
       processor: 'markdown/markdown'
     },
     {
-      // Check markdown `*.md` contains valid code blocks
-      // https://www.npmjs.com/package/eslint-plugin-markdown#user-content-advanced-configuration
-      files: ['**/*.md/*.{cjs,js,mjs}'],
+      files: [
+        '**/assets/javascript/**/*.{cjs,js,mjs}',
+        '**/docs/examples/**/*.{cjs,js,mjs}',
+        '**/*.md/*.{cjs,js,mjs}'
```

Comment:
Why do we have the `md` check files line up top and down here? I think maybe I don't fully understand how each of these override blocks are applied / relate to each other...